R/data_maggie.R

```diff
@@ -0,0 +1,464 @@
+#'  \item{carmelo2_post}{Team 2's CARMELO rating after the game}
+#'  \item{carmelo_prob1}{Team 1's probability winning based on CARMELO rating}
+#'  \item{carmelo_prob2}{Team 2's probability of winning based on CARMELO rating}
+#'  \item{score1}{Points scored by Team 1}
+#'  \item{score2}{Points scored by Team 2}
+#'}  
+#'@source See \url{https://projects.fivethirtyeight.com/nba-model/nba_elo.csv}
+
+
+
+#'The Complete History Of The NFL
+#'2017 NFL Predictions
+#'
+#'The raw data behind the story 
+#'"The Complete History of the NFL"
+#'\url{https://projects.fivethirtyeight.com/complete-history-of-the-nfl/}
+#'And our "2017 NFL Predictions"
+#'\url{https://projects.fivethirtyeight.com/2017-nfl-predictions/}
+#'
+#'@format a data frame with 16,007 rows representing games and 14 variables:
+#'\describe{
+#'  \item{date}{Date}
+#'  \item{season}{Season year, 1920-2018}
+#'  \item{neutral}{TRUE if the game was played on neutral territory, FALSE if not}
+#'  \item{playoff}{TRUE if the game was a playoff game, FALSE if not}
+#'  \item{team1}{The name of one participating team}
+#'  \item{team2}{The name of the other participating team}
+#'  \item{elo1_pre}{Team 1's Elo rating before the game}
+#'  \item{elo2_pre}{Team 2's Elo rating before the game}
+#'  \item{elo_prob1}{Team 1's probability of winning based on Elo rating}
+#'  \item{elo_prob2}{Team 2's probability of winning based on Elo rating}
+#'  \item{elo1_post}{Team 1's Elo rating after the game}
+#'  \item{elo2_post}{Team 2's Elo rating after the game}
+#'  \item{score1}{Points scored by Team 1}
+#'  \item{score2}{Points scored by Team 2}
+#'}
+#'@source See \url{https://projects.fivethirtyeight.com/nfl-api/nfl_elo.csv}
+
+
+
+#'How Every NFL Team’s Fans Lean Politically
+#'
+#'The raw data behind the story
+#'"How Every NFL Team’s Fans Lean Politically"
+#'\url{https://fivethirtyeight.com/features/how-every-nfl-teams-fans-lean-politically}
+#'
+#'Google Data
+#'@format a data frame with 207 rows representing designated market areas and 9 variables:
+#'\describe{
+#'  \item{dma}{Designated Market Area}
+#'  \item{nfl}{The percentage of search traffic in the media market region related to the NFL over the past 5 years}
+#'  \item{nba}{The percentage of search traffic in the region related to the NBA over the past 5 years}
+#'  \item{mlb}{The percentage of search traffic in the region related to the MLB over the past 5 years}
+#'  \item{nascar}{The percentage of search traffic in the region related to NASCAR over the past 5 years}
+#'  \item{cbb}{The percentage of search traffic in the region related to the CBB over the past 5 years}
+#'  \item{cfb}{The percentage of search traffic in the region related to the CFB over the past 5 years}
+#'  \item{trump_2016_vote}{The percentage of voters in the region who voted for Trump in the 2016 Presidential Election}
+#'  }
+#' @source Google Trends \url{https://g.co/trends/5P8aa}
+#' To convert data frame to tidy data (long) format, run:
```

Comment:
For any example to work, you'll need to add a `@example` roxygen tag. Also note that comments need to have `#`. So line 91 should look like:
```
#' # To convert data frame to tidy data (long) format, run:
```